addons/event/data/event_demo.xml

```diff
@@ -61,9 +63,48 @@
         <field eval="(DateTime.today()+ timedelta(days=100)).strftime('%Y-%m-%d 20:15:00')" name="date_begin"/>
         <field eval="(DateTime.today()+ timedelta(days=101)).strftime('%Y-%m-%d 00:30:00')" name="date_end"/>
         <field name="event_type_id" ref="event_type_data_physical"/>
-        <field name="address_id" ref="base.res_partner_1"/>
+        <field name="address_id" ref="event.res_partner_location_0"/>
         <field name="seats_availability">unlimited</field>
         <field name="stage_id" ref="event_stage_booked"/>
+        <field name="kanban_state">blocked</field>
+    </record>
+
+    <record id="message_event_1_0" model="mail.message">
+        <field name="model">event.event</field>
+        <field name="res_id" ref="event.event_1"/>
+        <field name="body"><![CDATA[<p>Hello Marc Demo,<br/>
```

Comment:
Do not use Cdata, use type="html" instead. That way you have a real syntax highlight in your linter :) .